spec/std/int_spec.cr

```diff
@@ -777,4 +777,11 @@ describe "Int" do
     65.unsafe_chr.should eq('A')
     (0x10ffff + 1).unsafe_chr.ord.should eq(0x10ffff + 1)
   end
+
+  it "#bit_length" do
```

Comment:
Test negative numbers? perhaps `BigInt` too but that's not neccesary if it doesn't work (but you should move it to only primitive ints if it doesn't)

Reply:
I added one for negative numbers. I also added a fallback for non Int::Primitive types.
Thanks!